scripts/delete-hpa-test-pods.sh

```diff
@@ -5,7 +5,7 @@ SCRIPT_DIR=$(dirname "$0")
 source $SCRIPT_DIR/init-env.sh
 
 # Delete the hpa 
-oc delete hpa -n ${TNF_EXAMPLE_CNF_NAMESPACE}
+oc delete hpa hpa -n ${TNF_EXAMPLE_CNF_NAMESPACE}
```

Comment:
naming an hpa service hpa is not very intuitive. I would prefer simple name deployment. 
partner / test-hpa